overlays/nodejs/serverless/function.ts

```diff
@@ -12,41 +12,27 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-import * as crypto from "crypto";
 import * as pulumi from "@pulumi/pulumi";
-import { Role, RolePolicyAttachment } from "../iam";
+import { Role } from "../iam";
 import * as lambda from "../lambda";
-import { Overwrite } from "../utils";
-import { ARN } from "../arn";
 
-/**
- * Context is the shape of the context object passed to a Function callback.
- */
-export interface Context {
-    callbackWaitsForEmptyEventLoop: boolean;
-    readonly functionName: string;
-    readonly functionVersion: string;
-    readonly invokedFunctionArn: string;
-    readonly memoryLimitInMB: string;
-    readonly awsRequestId: string;
-    readonly logGroupName: string;
-    readonly logStreamName: string;
-    readonly identity: any;
-    readonly clientContext: any;
-    getRemainingTimeInMillis(): string;
-}
+export type Context = lambda.Context;
```

Comment:
all this code is effectively deprecated in favor of the corresponding types in aws.lambda.XXX.  However, for back compat, htey have been marked deprecated, but kept here.